test/snapshot/__snapshots__/constantstatement.test.js.snap

```diff
@@ -53,6 +57,8 @@ Program {
             "kind": "identifier",
             "name": "CONSTANT",
           },
+          "nullable": null,
+          "type": undefined,
```

Comment:
If we do not want those to be part of the AST, we need to completly decouple the ClassConstant object from the Constant object.